app/Http/Controllers/UserController.php

```diff
@@ -75,10 +75,10 @@ public function show($userid){
         ]);
     }
 
-    public function admin($userid){
+    public function admin($userid) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->